src/Pokémon.java

```diff
@@ -0,0 +1,52 @@
+package src;
+
+public abstract class Pokémon {
+
+    private String sound;
+    private String food;
+
+
+    public Pokémon(String sound, String food) {
+        this.food = food;
+        this.sound = sound;
+
+    }
+
+    public void setSound(String sound) {
+        if (sound != null && !sound.isEmpty()) {
+            this.sound = sound;
+        }
+    }
+
+    public String getSound() {
+        return sound;
+    }
+
+    public String getFood() {
+        return food;
+    }
+
+    public void setFood(String food) {
+        this.food = food;
+    }
+
+    public void pokemonBasic(String sound, String food) {
+        sound = getSound();
+        food = getFood();
+
+        if (sound.equals("theme") && food.equals("hamburger")) {
+            System.out.println("All Pokemon eats " + food + " and starts withe the sound " + sound + ".");
+        } else {
+            System.out.println("this belongs to an other Pokemon!");
+        }
+    }
+
+    public static void Pikachu(String sound, String food) {
```

Comment:
In deze methode ga je het nut van een parameter voorbij. Je geeft twee parameters op, `sound` en `food`. Vervolgens negeer je welke waarde je daar invult bij de aanroep (in main), want je overschrijft het met `"baby sounds"` en `"fruits"` en daar print je vervolgens een regel mee uit. 
Het maakt dus niet uit of ik deze methode aanroep als `Pikachu("banaan", "appel")` of `Pikachu("Hogeschool","Novi")`, want het printstatement geeft altijd hetzelfe resultaat. 
Je kunt in deze methode de parameters dus gewoon weg laten, zodat je de methode makkelijk kunt aanroepen als `Pikachu()`